app/models/user.rb

```diff
@@ -1,5 +1,9 @@
 class User < ApplicationRecord
   has_secure_password
-  validates :name,:email,:password, presence: true
+
+  #VALIDATIONS
+  validates :name, :email, :password, :contact_number, presence: true
   validates :password, confirmation: { case_sensitive: true }
+  validates :email, uniqueness: true
```

Comment:
Suggestion :You can also use regex for email validation !
Up to you to implement it or not !